api/bin/www

```diff
@@ -20,12 +21,16 @@ app.set("port", port);
  * Connect to MongoDB
  **/
 
-var mongoDbUrl =
-  process.env.MONGODB_URL || "mongodb://0.0.0.0/recipe-generator";
-mongoose.connect(mongoDbUrl, {
-  useNewUrlParser: true,
-  useUnifiedTopology: true,
-});
+var mongoDbUrl = process.env.DB_CONNECT;
```

Comment:
The .env file cannot be found in the repo.
Technically it should be linked into the .gitignore so that the file cannot be shown in public because it contains sensitive information about the database, but in this scenario I don't have it stored

Reply:
I think the .env is in gitignore already, which is why it's not in repo. Everyone will have to create the file themselves locally